src/runtime/objmodel.cpp

```diff
@@ -3266,7 +3274,7 @@ void rearrangeArguments(ParamReceiveSpec paramspec, const ParamNames* param_name
         Box* ovarargs = BoxedTuple::create(unused_positional.size(), &unused_positional[0]);
         getArg(varargs_idx, oarg1, oarg2, oarg3, oargs) = ovarargs;
     } else if (unused_positional.size()) {
-        raiseExcHelper(TypeError, "%s() takes at most %d argument%s (%ld given)", func_name, paramspec.num_args,
+        raiseExcHelper(TypeError, "%s() takes exactly %d argument%s (%ld given)", func_name, paramspec.num_args,
```

Comment:
I'm not sure that either of these error messages is quite correct; maybe it should be `paramspec.num_defaults ? "at most" : "exactly"`